src/Trixi.jl

```diff
@@ -46,6 +46,11 @@ using TimerOutputs: TimerOutputs, @notimeit, TimerOutput, print_timer, reset_tim
 @reexport using UnPack: @unpack
 using UnPack: @pack!
 
+# finite difference SBP operators
+using SummationByPartsOperators: AbstractDerivativeOperator, DerivativeOperator, grid
+import SummationByPartsOperators: integrate, left_boundary_weight, right_boundary_weight
```

Comment:
I'm wondering... should we maybe only import the SBP package and explicitly use `SummationByParts.integrate` to show that we're extending existing functions in the code? That might help to prevent accidental changes to function API by unsuspecting developers

Reply:
But that would make it quite cumbersome to use both SummationByPartsOperators.jl and Trixi.jl in the same project since very call to `integrate` would need to be prefixed...

Reply:
But we could be `using` it, right? Then it would still be extensible this way but no prefixing necessary? But maybe I'm missing something...

Reply:
We also export `integrate` from Trixi, so there will be a name-clash if we don't `import integrate` here

Reply:
OK, so does this then mean that with this change, all methods named `integrate` will become methods of the function `integrate` originally defined in `SBPOps.jl`? Does this have any downsides/visible effects on us or our users? E.g., will people still be able to do `import Trixi: integrate` to add new methods to *our* integrate methods?

Reply:
> does this then mean that with this change, all methods named `integrate` will become methods of the function `integrate` originally defined in `SBPOps.jl`?

Yes

> Does this have any downsides/visible effects on us or our users?

I don't think so - at least I don't have any examples in mind

> will people still be able to do `import Trixi: integrate` to add new methods to _our_ integrate methods?

Yes (although technically speaking it's the same function as `integrate` in SummationByPartsOperators.jl)

Reply:
Ok. It still feels somewhat iffy to me, but then again, I am not nearly as familiar with the Julia type and extension rules as you are, so if you're ok with it, I have no objections :-)

Reply:
The basic question is whether we use the same concept - which we do. Thus, we should use the same verb `integrate`. There are even packages out there that basically just define such a common verb that is extended later by lots of other packages, e.g., [CommonSolve.jl](https://github.com/SciML/CommonSolve.jl).

Reply:
Thanks for the explanation!